classify.py

```diff
@@ -24,10 +24,10 @@
 from ml_tools.trackextractor import TrackExtractor, Track, Region
 
 
-DEFAULT_BASE_PATH = "c:/cac"
+DEFAULT_BASE_PATH = "/Users/clare/cacophony/model"
```

Comment:
Pretty sure this isn't a good choice :)

There probably shouldn't be a default base path at all.

Reply:
I totally agree.   We really need to "productionise" the whole classifier module - starting with configuration files.